utility/src/main/java/org/oppia/android/util/parser/html/LiTagHandler.kt

```diff
@@ -96,7 +98,7 @@ class LiTagHandler(
           ListItemLeadingMarginSpan.OlSpan(
             context,
             indentation,
-            "${displayLocale.run { (mark.number).toHumanReadableString(mark.number) }}.",
+            "${displayLocale.run { toHumanReadableString(mark.number) }}.",
```

Comment:
Do you still need to use ``run`` or can you just call the method directly on the locale? Ditto for the updated tests in DisplayLocaleImplTest.

Reply:
Done.